catalog/ui/search-ui/search-htmltransformer/src/main/java/org/codice/ddf/catalog/transformer/html/RecordViewHelpers.java

```diff
@@ -38,16 +44,32 @@
 
 public class RecordViewHelpers {
 
-    public static final SimpleDateFormat ISO_8601_DATE_FORMAT;
-
     private static final Logger LOGGER = LoggerFactory.getLogger(RecordViewHelpers.class);
 
+    private static TransformerFactory transformerFactory;
+
     private static DocumentBuilderFactory documentBuilderFactory;
 
+    private static final ThreadLocal<SimpleDateFormat> SIMPLE_DATE_FORMAT_THREAD_LOCAL =
```

Comment:
Unless we know that there would be a performance problem with creating a new `SimpleDateFormat` object every time or synchronizing on the object, creating a `ThreadLocal` for this seems a little extreme.